### PR TITLE
Manual part of reorganization

### PR DESCRIPTION
**Pull request by diana-lakatos**
I did the manual part of the reorganization (URL changes). Please review and merge before deploying the reorganization to prod. 

* edited navigation
* edited home page description and cards
See: 
https://diana-test.staging-california.near-me.com

* edited section index pages (How Platform OS Works, Get Started, Tutorials), will edit these further, now I just removed cards and updated description
See: 
https://diana-test.staging-california.near-me.com/how-platformos-works
https://diana-test.staging-california.near-me.com/get-started
https://diana-test.staging-california.near-me.com/tutorials

* fixed topics, folders, and navigation links that weren't included in the JSON map, so weren't automatically moved

### CODE REVIEW

**Comment by pavelloz** [comment]
https://diana-test.staging-california.near-me.com/api-reference/liquid-filters-platformos - i think this page doesnt belong to APIs - its a concept page, short and probably needs to be in some kind of getting started, not among autogenerated docs

**Comment by pavelloz** [comment]
I know it doesnt have anything to do with this reorganization, but Search section

https://diana-test.staging-california.near-me.com/tutorials/search/search

is really about pulling transactables/customizations/users from graphql and filtering/sorting them in a correct way.

We should probably reword it and create section of graphql tutorials where we show examples of the most important features when it comes to pulling data from the database - both, small portions (ie. one user) and big (ie. all customizations of given type, sorted by creation time, filtered by some property).

**Comment by pavelloz** [comment]
Same as above, https://diana-test.staging-california.near-me.com/tutorials/blog/installing-blog-module

This is something different than tutorial about how to use the platform (until we have blog module that is in our new module format) -- IMO we need to separate documentation from modules and only teach users how to use the high level of it - meaning, this article:
https://github.com/mdyd-dev/nearme-documentation/pull/249 

and every module should be released (and most of them open sourced on github for example) & documented within its own readme file, as they are independent from the platform after they have been released and) 

In short, i would hide the blog section altogether.

**Comment by pavelloz** [comment]
https://diana-test.staging-california.near-me.com/tutorials/custom-model-types/custom-model-types - this doesnt look like tutorial at all. it looks like a intro to a concept

**Comment by pavelloz** [comment]
https://diana-test.staging-california.near-me.com/tutorials/migrations/migrating-data - this is also not a tutorial per se.

Maybe we need marketplace-kit section in docs where we explain every command in detail

tutorials/marketplace-kit/migrations
tutorials/marketplace-kit/logs

Then we would move those topis as well there:

https://diana-test.staging-california.near-me.com/tutorials/qa/creating-marketplace-kit-logs
https://diana-test.staging-california.near-me.com/tutorials/qa/developing-graphql-queries-using-marketplace-kit-gui

This would open up a way to write a pretty full article about recommended workflow.

add envs, run sync, run logs, run gui serve, develop your gql queries, etc.

**Comment by pavelloz** [comment]
I think concept pages like this:
https://diana-test.staging-california.near-me.com/tutorials/activity-feeds/activity-feeds

should have different titles - like Introduction or something.

image

^ this doesnt really tell me what i will be reading if i click on it

**Comment by pavelloz** [comment]
https://diana-test.staging-california.near-me.com/tutorials/translations/translations - i would hide this section for the time being - looks incomplete and has no examples, etc.

**Comment by pavelloz** [comment]
https://diana-test.staging-california.near-me.com/api-reference/graphql/graphql-glossary - page that is not API description - probably belongs into getting started with graphql section

**Comment by diana-lakatos** [comment]
> https://diana-test.staging-california.near-me.com/api-reference/liquid-filters-platformos - i think this page doesnt belong to APIs - its a concept page, short and probably needs to be in some kind of getting started, not among autogenerated docs

Yes, I wanted to discuss this in the next round, whether we can put this section right before the auto-generated content, or move it somewhere else. 

> In short, i would hide the blog section altogether.

Yes, all of these topics are planned to be added as the user documentation of the blog Module on the Module Marketplace. And we will write a new use case where we explain how we implemented the blog module (not how to use it). 

> https://diana-test.staging-california.near-me.com/tutorials/translations/translations - i would hide this section for the time being - looks incomplete and has no examples, etc.

We will have to think about hiding incomplete sections or topics, because on one hand, they can be frustrating for users, but on the other hand they are indicators that we are planning content there and also welcome contributions. I'd like to discuss this with the team and Kata.

**Comment by pavelloz** [comment]
👍
